core/src/test/java/io/nixer/nixerplugin/core/detection/registry/FailedLoginRatioRegistryTest.java

```diff
@@ -22,31 +23,41 @@
 
     private FailedLoginRatioRegistry registry;
 
+    private final Instant baseInstant = Instant.ofEpochSecond(600);
+
+    @BeforeEach
+    void setUp() {
+        registry = new FailedLoginRatioRegistry(Duration.ofMinutes(20), nowSource);
```

Comment:
`Duration.ofMinutes(20)` would better be extracted to constant, e.g. `ACTIVATION_TIMEOUT`, and also used in `activaitonTimeout` test below.